aries-site/src/pages/components/layer/center-layer.mdx

```diff
@@ -0,0 +1,68 @@
+
+<Example plain>
+  <CenterLayerAnatomy />
+</Example>
+
+| Label | Region             | Purpose                                                          | Required?                                  | Notes                                                                                                                                                                                                                                |
+| ----- | ------------------ | ---------------------------------------------------------------- | ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Close button**   | Closes the layer.                                                | Required on layers with no footer actions. | Use "FormClose" icon.                                                                                                                                                                                                                |
+| **2** | **Footer actions** | Allows the user to confirm, apply, or cancel the layer's action. | Required on actionable layers.             | The actions should be right-aligned with primary action on the right. Right-alignment signals progress and forward momentum. One action should be a "Cancel" button that closes the layer, returning the user to the previous state. |
+
+See [layer closing behavior](/components/layer#layer-closing-behavior) for details.
+
+<Notification
+  status="info"
+  message="A layer should either have actions in the footer or a close button in the header, not both."
+  width="large"
+  margin={{ bottom: 'medium' }}
+/>
+
+## Examples
+
+### Destructive confirmation
+
+<Notification
+  status="info"
+  message="Additional destructive confirmation patterns differing based off of level of severity are coming soon."
+  width="large"
+  margin={{ bottom: 'medium' }}
+/>
+
+For highly destructive actions, have the user type a string that matches the name or path of the object to be deleted. This approach ensures the user is aware of the consequences of the action.
+
+<Example
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContentDrivenLayout.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/ModalDialog/ModalDialog.js',
+  ]}
+  github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/templates/content-layouts/ContentDrivenLayout.js"
+  showResponsiveControls={['fullScreen']}
+  screenContainer
+  height="650px"
+  width="100%"
+>
+  <ContentDrivenLayout />
+</Example>
+
+### Confirming next steps
```

Comment:
This looks identical to the previous section.

Reply:
Thanks, updated.